src/main/java/net/pms/dlna/DLNAOrgFlags.java

```diff
@@ -0,0 +1,564 @@
+package net.pms.dlna;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * This class represents the {@code DLNA.ORG_FLAGS} parameter.
+ *
+ * @author Nadahar
+ */
+public class DLNAOrgFlags {
+
+	/** The first/most significant bits/flags. */
+	private final long high;
+
+	/** The last/least significant bits/flags. */
+	private final long low;
+
+	/**
+	 * A {@link DLNAOrgFlags} instance that represents the implied flags where
+	 * all flags are 0 and results in an empty parameter string.
+	 */
+	public static final DLNAOrgFlags IMPLIED = new DLNAOrgFlags(0, 0);
+
+	/**
+	 * Calculates the "effective" flags for a {@link DLNAOrgFlags} instance by
+	 * applying DLNA conditional rules for some flags. {@code null} can be
+	 * passed to get the flags that represent an omitted {@code DLNA.ORG_FLAGS}
+	 * parameter.
+	 *
+	 * @param flags the {@link DLNAOrgFlags} instance to process.
+	 * @param mediaType the {@link MediaType} of the media for which
```

Comment:
It seems so... something must have happened, a line deleted, because I remember that I wrote that sentence.